backend/src/server.js

```diff
@@ -27,5 +28,11 @@ app.post("/", (req, res) => {
   res.status(200).json({ msg: "Backend Post Working" });
 });
 
+app.get("/1", (req, res) => {
```

Comment:
what is this route for? if only sample, maybe put something beneficial in the future like 404 page

Reply:
For testing the Error middleware

Reply:
@IanUrriza10 my suggestion would be to put a comment so it's understandable to be removed in the future or maybe change this route to 404 page to be used later as well

Reply:
Noted po. I have converted it to catch all types of requests and send a 404 not found if needed.